assets/js/util.js

```diff
@@ -119,6 +119,27 @@ $(document).ready(function () {
     $('.modal').on('hide.bs.modal', function () {
         $('a[data-target=#' + $(this).attr('id') + ']').parents('li').removeClass('active');
     });
+
+    $(window).scroll(function () {
+        if($(window).scrollTop() > BACK_TO_TOP_ACTIVATION_HEIGHT) {
+            $('#backToTop')
+            .removeClass('hidden-xs hidden-sm hidden-md hidden-lg')
```

Comment:
Huh, I'm surprised that TravisCI/Hound, i.e. ESLint doesn't catch this. I might have my rules configured incorrectly. Regardless, L126 and L127 here should be indented (just as 131 and 132).